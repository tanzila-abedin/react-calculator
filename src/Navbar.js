import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav className="navbar">
    <div className="links">
      <Link to="/">Home</Link>
      <Link to="/Wrapper">Calculator</Link>
      <Link to="/Quote">Quotes</Link>
    </div>
  </nav>
);

export default Navbar;
