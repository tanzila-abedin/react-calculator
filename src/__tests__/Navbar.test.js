import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router-dom';
import Navbar from '../Navbar';

test('Navbar snapshot testing', () => {
  const tree = renderer
    .create(
      <StaticRouter>
        <Navbar />
      </StaticRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
