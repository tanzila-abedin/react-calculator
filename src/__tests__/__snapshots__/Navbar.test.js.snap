// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot testing 1`] = `
<nav
  className="navbar"
>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/Quote"
      onClick={[Function]}
    >
      Quotes
    </a>
  </div>
</nav>
`;
